pgjdbc/src/test/java/org/postgresql/test/jdbc42/GetObject310BinaryTest.java

```diff
@@ -9,10 +9,6 @@
 
 public class GetObject310BinaryTest extends GetObject310Test {
 
-  public GetObject310BinaryTest(String name) {
```

Comment:
It would be called.

I removed the GetObject310BinaryTest  and made the GetObject310Test as  parameterized.